.github/workflows/ibis-backends.yml

```diff
@@ -447,3 +447,90 @@ jobs:
         with:
           name: ${{ matrix.backend }}-${{ matrix.python-version }}
           path: junit.xml
+
+  test_datafusion:
+    name: DataFusion ${{ matrix.datafusion-version }} ubuntu-latest python-${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        datafusion-version:
+          - master
+          - latest
```

Comment:
Do we need to test both or would `latest` suffice?

Reply:
I'd prefer to test using both since we'll need to develop additional features to the datafusion python bindings to improve the support for ibis operators.

Reply:
SGTM.